hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -106,12 +152,12 @@ private Schema getTestSchema() {
   public void testCreateTableBuilder() throws Exception {
     Schema schema = getTestSchema();
     PartitionSpec spec = PartitionSpec.builderFor(schema).bucket("data", 16).build();
-    TableIdentifier tableIdent = TableIdentifier.of(DB_NAME, "tbl");
+    TableIdentifier tableIdent = TableIdentifier.of(HIVE_DB_NAME, "tbl");
     String location = temp.resolve("tbl").toString();
 
     try {
       Table table =
-          catalog
+          catalog()
```

Comment:
If you keep a local variable `HiveCatalog catalog` and initialize it in `before()`, you don't have to change to a method call everywhere;